src/global/core/shared/normalize.scss

```diff
@@ -0,0 +1,273 @@
+/*! modern-normalize v1.1.0 | MIT License | https://github.com/sindresorhus/modern-normalize */
```

Comment:
This provides a normalized app shell since it is globally referenced (for the shared mode). For the single component usage (default mode) this might be a bit of bloat but normalizes the component and childeren (could be slots) too.